jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -6256,6 +6262,250 @@
+						},
+						"description": "Creating a new JVM for JDK upgrade testing."
+					},
+					"response": []
+				},
+				
+				{
+					"name": "upgrade JDK- Negative Test case",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 500\"] = responseCode.code === 500;",
+											"",
+									"var jsonData = JSON.parse(responseBody);",
+									"tests[\"Assert 500 error\"] = jsonData.status === 500;",
+
+									"tests[\"Assert -JVM not in STOPPED state- error\"] = jsonData.message === \"The remote JVM: \"+pm.variables.get(\"JDKUpgradeJvmName\")+\" must be in STOPPED state before attempting to upgrade the JDK of the JVM\";"
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{url}}/v1.0/jvms/{{JDKUpgradeJvmName}}/jdk",
+						"method": "PUT",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"description": "This webservices tries to upgrade the JDK version of the JVM which is not yet generated."
+					},
+					"response": []
+				},
+				{
+					"name": "generate jvm",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 200\"] = responseCode.code === 200;"
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{url}}/v1.0/jvms/{{JDKUpgradeJvmName}}/conf",
+						"method": "PUT",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"description": "Generating the JDK upgrade JVM. "
+					},
+					"response": []
+				},
+				{
+					"name": "upgrade JDK- Negative Test case",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 500\"] = responseCode.code === 500;",
+											"",
+									"var jsonData = JSON.parse(responseBody);",
+									"tests[\"Assert 500 error\"] = jsonData.status === 500;",
+
+									"tests[\"Assert No setenv.bat error\"] = jsonData.message === \"The setenv.bat file does not exist as a resource for JVM: \"+pm.variables.get(\"JDKUpgradeJvmName\")+\". Please create a setenv.bat resource for the JVM in order to set the JAVA_HOME environment variable.\"|| jsonData.message === \"The setenv.sh file does not exist as a resource for JVM: \"+pm.variables.get(\"JDKUpgradeJvmName\")+\". Please create a setenv.sh resource for the JVM in order to set the JAVA_HOME environment variable.\";"
```

Comment:
This line is way too long.
Break up each assertion message into its own variable: one for windows and one for linux, and then created the conditional check from those variables.

Reply:
Use the targetOs variable to determine if the test is for windows or linux:
https://github.com/cerner/jwala/blob/master/jwala-integration-test/src/test/postman/hostUserPassword_Environment.js#L10

https://github.com/cerner/jwala/blob/master/jwala-integration-test/src/test/postman/hostUserPassword_Environment.js#L95

https://github.com/cerner/jwala/blob/master/jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json#L1037